### PR TITLE
models created and .gitignore modeified.

### PR DESCRIPTION
**Pull request by quratulain25**

### CODE REVIEW

File: todo_list/models.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
-# Create your models here.
+
+class Page(models.Model):
+    title = models.CharField(max_length=10)
```
Comment (ifzal) [inline todo_list/models.py]: @quratulain25 Add **create_at** and **modified_at** date time fields for every model where entries can be changed any time.

**Review by ifzal — COMMENTED** [review]
A little modification, rest is good.

File: todo_list/models.py [hunk 2]
```diff
@@ -1,3 +1,21 @@
 from django.db import models
 
-# Create your models here.
+
+class Page(models.Model):
+    def __str__(self):
```
Comment (ifzal) [inline todo_list/models.py]: @quratulain25 While creating models make sure fields come before any method.

File: todo_list/models.py [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+        return self.title
+
+    title = models.CharField(max_length=30)
```
Comment (ifzal) [inline todo_list/models.py]: @quratulain25 Do you think 30 char length is sufficient? Also while determining char length use 2^x formula.

File: serendipity/settings.py [hunk 4]
```diff
@@ -38,7 +38,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'users.apps.UsersConfig',
+    'todo_list.apps.TodoListConfig'
```
Comment (ifzal) [inline serendipity/settings.py:41]: @quratulain25 I think this app was directly added to master branch so I can't review TodoListConfig

File: .gitignore [hunk 5]
```diff
@@ -1,3 +1,10 @@
+*~
+/.vscode
+myvenv
```
Comment (ifzal) [inline .gitignore]: @quratulain25 For a good overview or a better gitignore you should visit https://github.com/github/gitignore

**Review by ifzal — CHANGES_REQUESTED** [review]

File: todo_list/models.py (hunk 3 above)
Comment (quratulain25) [reply todo_list/models.py]: @ifzal , what is x?

**Pull request merged**